build.gradle

```diff
@@ -92,7 +92,7 @@ dependencies {
 task testIntegration(type: Test) {
     include '**/*IntTest*'
     useJUnitPlatform()
-    minHeapSize = "1024m"
+    minHeapSize = "128m"
```

Comment:
Not sure this makes any difference but the tests are currently stable so :crossed_fingers: